include/seqan3/io/stream/concept.hpp

```diff
@@ -49,11 +49,12 @@
 namespace seqan3
 {
 
-/*!\addtogroup io
+/*addtogroup stream
```

Comment:
why not `/*! \addtogroup` to make it doxygen-visible?

Reply:
This is still not resolved (missing !)